uc.c

```diff
@@ -242,6 +244,10 @@ static uc_err uc_init(uc_engine *uc)
         uc->reg_reset(uc);
     }
 
+    uc->context_content = UC_CTL_CONTEXT_CPU;
+
+    uc->unmapped_regions = g_array_new(false, false, sizeof(MemoryRegion*));
```

Comment:
I fail to find the location to free this memory. Do I miss something?

Reply:
No, I missed the free.